utils/make-sqlite.py

```diff
@@ -89,7 +97,14 @@
     ruleset_id = c.lastrowid
```

Comment:
Is there a reason to keep these lines of code around? If not they should be deleted. Also the sqlite cursor and the `c` variable should probably be removed.